### PR TITLE
KK-277 | Child page event listing

### PR DESCRIPTION
**Pull request by jpsalo**
Draft notes:
- [ ] There's a bunch of TODOs which don't affect the functionality but will be resolved before merge
- [ ] Image is mercilessly resized to 200x200 for now
- [x] Typings require reiteration
- [x] It's mobile first but not mobile-optimized
- [ ] event duration determines occurrence duration - now it's hard-coded to 30 minutes
- [ ] calendar icon is missing
- [ ] enrolments return past events but it will be fixed in the backend
- [ ] tests for empty list cases are missing

localhost_3000_fi_profile_child_Q2hpbGROb2RlOjliODJiOTQxLWZlMWMtNGQwMy1hMTU4LWZmOTM5YmI3NWIwNw== (1)

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -1,2 +1,2 @@
-REACT_APP_API_URI="https://kukkuu.test.kuva.hel.ninja/graphql"
```
Comment (midori0507) [inline .env.development]: Uhm maybe no, keep it currently

File: package.json [hunk 2]
```diff
@@ -69,6 +69,7 @@
     "test": "react-scripts test --watchAll=false",
     "test:debug": "react-scripts --inspect-brk test --runInBand --no-cache"
   },
+  "proxy": "http://localhost:8081",
```
Comment (midori0507) [inline package.json]: why this one defined here ?

File: src/common/components/card/card.module.scss [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    align-items: center;
+    margin: 0;
+    padding: 0;
+  }
+  background: $white;
+  & + & {
+    margin-top: 20px;
+  }
+  margin: 0 -1rem;
+  padding: 1rem 1rem;
```
Comment (midori0507) [inline src/common/components/card/card.module.scss]: just padding: 1rem  ? 😄

File: src/common/components/card/card.module.scss [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  }
+  >:last-child {
+    margin-bottom: 0;
+  }
+  margin: 20px 0;
+}
+
+.end {
+  @include respond-above(sm) {
+    margin-right: 20px;
```
Comment (midori0507) [inline src/common/components/card/card.module.scss]: be careful with duplicated free value like this, we discussed yesterday

File: src/domain/child/queries/ChildQueries.ts [hunk 5]
```diff
@@ -40,6 +40,63 @@ export const childByIdQuery = gql`
...
+              }
+            }
+          }
+        }
+      }
+      availableEvents {
+        edges {
+          node {
+            id
+            translations {
```
Comment (badrange) [inline src/domain/child/queries/ChildQueries.ts]: you can get the name directly here if you have an up to date backend
No need to fiddle with translations!

File: src/common/components/card/Card.tsx [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+  primaryActionText,
+  title,
+}) => {
+  const { t } = useTranslation();
+
+  return (
+    <div className={styles.wrapper}>
+      <div className={styles.start}>
+        {/* TODO: alt, size */}
+        <img src={image} alt={'alt'} width="200" height="200" />
```
Comment (badrange) [inline src/common/components/card/Card.tsx]: For mobile I don't think you need to define the width, you can just let it fill the screen? 

And if you don't define height either you can allow the editors to choose the image aspect ratio (not sure if that is a good idea)

File: .env.development [hunk 7]
```diff
@@ -1,2 +1,2 @@
...
+REACT_APP_API_URI="ttps://kukkuu.test.kuva.hel.ninja/graphql"
```
Comment (badrange) [inline .env.development]: I think an H is missing :D

**Pull request merged**